gapis/api/vulkan/command_buffer_rebuilder.go

```diff
@@ -55,6 +55,8 @@ func allocateNewCmdBufFromExistingOneAndBegin(
 	modelCmdBuf VkCommandBuffer,
 	s *api.GlobalState) (VkCommandBuffer, []api.Cmd, []func()) {
 
+	a := s.Arena // TODO: Should this be a seperate temporary arena?
```

Comment:
I think it could be, but definitely not for this MR. It's probably best to avoid that kind of optimization for now. I worry that there might be some tricky business with allocations.

Reply:
Completely agree. This is going to require a lot of iteration.